clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10982,6 +10982,8 @@ def warn_sycl_implicit_decl
 def warn_sycl_restrict_recursion
     : Warning<"SYCL kernel cannot call a recursive function">,
       InGroup<SyclStrict>, DefaultError;
+def warn_sycl_implicit_attr_usage : Warning <
+  "%0 attribute cannot be used explicitly">, InGroup<IgnoredAttributes>;
```

Comment:
What do you mean by this?  Why give this a spelling at all if you don't want users using it?